projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -526,7 +527,33 @@ const VpBlock = props => {
 	return (
 		<figure { ...blockProps }>
 			<div className="wp-block-embed__wrapper">
-				<SandBox html={ html } scripts={ scripts } type={ videoPressClassNames } />
+				<ResizableBox
+					enable={ {
+						top: false,
+						bottom: false,
+						left: true,
+						right: true,
+					} }
+					maxWidth="100%"
+					size={ { width: maxWidth } }
+					style={ { margin: 'auto' } }
+					/* eslint-disable no-unused-vars */
+					onResizeStop={ ( event, direction, elem, delta ) => {
```

Comment:
I wish there was a way to replace unused vars with `_` like in some other languages 😄 . Maybe there is a plugin to do that? But definitely outside the scope of this PR.

One thing that could work though is if you move the function out into a const outside of this return block and call it something like `handleResize`, it doesn't look like the linter cares that there are unused vars in those types of functions. And it will keep the `ResizableBox` markup small and easier to read.